src/components/tx/modals/NewTxModal/ReplacementModal.tsx

```diff
@@ -0,0 +1,121 @@
+import {
+  Button,
+  DialogContent,
+  SvgIcon,
+  Tooltip,
+  Typography,
+  Stepper,
+  Step,
+  StepLabel,
+  DialogActions,
+  Grid,
+} from '@mui/material'
+
+import ModalDialog from '@/components/common/ModalDialog'
+import InfoIcon from '@/public/images/notifications/info.svg'
+import RocketIcon from '@/public/images/transactions/rocket.svg'
+import CheckIcon from '@mui/icons-material/Check'
+import DeleteIcon from '@/public/images/common/delete.svg'
+import { SendTokensButton, SendNFTsButton } from './TxButton'
+
+import css from './styles.module.css'
+
+const wrapIcon = (icon: React.ReactNode) => <div className={css.circle}>{icon}</div>
+
+const steps = [
+  {
+    label: 'Create new transaction with same nonce',
+    icon: <div className={css.redCircle} />,
+  },
+  {
+    label: 'Collect confirmations from owners',
+    icon: wrapIcon(<CheckIcon fontSize="small" color="border" />),
+  },
+  {
+    label: 'Execute replacement transaction',
+    icon: wrapIcon(<SvgIcon component={RocketIcon} inheritViewBox fontSize="small" color="border" />),
+  },
+  {
+    label: 'Initial transaction is replaced',
+    icon: wrapIcon(<SvgIcon component={DeleteIcon} inheritViewBox fontSize="small" color="border" />),
+  },
+]
+
+const ReplacementModal = ({
+  txNonce,
+  onClose,
+  onTokenModalOpen,
+  onNFTModalOpen,
+  onRejectModalOpen,
+}: {
+  txNonce: number
+  onClose: () => void
+  onTokenModalOpen: () => void
+  onNFTModalOpen: () => void
+  onRejectModalOpen: () => void
+}) => {
+  return (
+    <ModalDialog open dialogTitle={`Replace transaction with nonce ${txNonce}`} onClose={onClose}>
+      <DialogContent className={css.container}>
+        <Typography variant="h5" mb={1}>
+          Need to replace or discard this transaction?
+        </Typography>
+        <Typography variant="body1" textAlign="center">
+          A signed transaction cannot be removed but it can be replaced with a new transaction with the same nonce.
+        </Typography>
+        <Stepper alternativeLabel className={css.stepper}>
+          {steps.map(({ label }) => (
+            <Step key={label}>
+              <StepLabel StepIconComponent={({ icon }) => steps[Number(icon) - 1].icon}>
+                <Typography variant="body1" fontWeight={700}>
+                  {label}
+                </Typography>
+              </StepLabel>
+            </Step>
+          ))}
+        </Stepper>
+      </DialogContent>
+      <DialogActions className={css.container}>
+        <Grid container alignItems="center" justifyContent="center" spacing={3}>
+          <Grid item xs={12}>
+            <Typography variant="body2" textAlign="center" fontWeight={700}>
+              Select the type of replacement transaction
+            </Typography>
+          </Grid>
+          <Grid item xs>
+            <SendTokensButton onClick={onTokenModalOpen} sx={{ mb: 1, maxWidth: '320px !important' }} />
```

Comment:
This doesn't seem to do anything even when I remove it. Why do we need to limit the width of this button and not the others?

Reply:
Good catch! It's a leftover from testing that I forgot to remove.